lib/tasks/user.rake

```diff
@@ -6,7 +6,8 @@ namespace :usasearch do
         user.set_approval_status_to_not_approved
         user.save!
         Emailer.user_approval_removed(user).deliver_now
-        NutshellAdapter.new.new_note(user, 'This user is no longer associated with any sites, so their approval status has been set to "not_approved".')
+        note = "User #{user.id}, #{user.email}, is no longer associated with any sites, so their approval status has been set to \"not_approved\"."
+        Rails.logger.info(note)
```

Comment:
tiny nit pick but you can do 'not_approved' instead of \"not_approved\" . looks cleaner.  :)